clang/docs/ReleaseNotes.rst

```diff
@@ -717,6 +717,8 @@ Bug Fixes to C++ Support
 - Clang now uses valid deduced type locations when diagnosing functions with trailing return type
   missing placeholder return type. (#GH78694)
 - Fixed a bug where bounds of partially expanded pack indexing expressions were checked too early. (#GH116105)
+- Clang now uses the location of the begin of the member expression for ``CallExpr``
```

Comment:
It would fit better under `bug fixes to ast handling`, right?

Reply:
@zyn0217 I've moved to `bug fixes to ast handling`